ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master__packetcheck.yaml

```diff
@@ -0,0 +1,31 @@
+build_root:
```

Comment:
This is not strictly needed for running the test, not sure if ci-operator requires it in any case

Reply:
I will try removing it.